kobotoolbox/kpi
# [TASK-1082] Add `mmo_override` field to Organization model with admin control

## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [x] Run `./python-format.sh` to make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/main/CONTRIBUTING.md)
5. [x] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes
8. [x] Create a testing plan for the reviewer and add it to the Testing section

## Description

This update introduces the ability to mark an organization as a `multi-member organization` (MMO) via the Django Admin interface. A new field, `mmo_override`, has been added to the `Organization` model, allowing superusers to manually activate or deactivate the multi-member status for any organization. This feature complements the existing functionality that automatically sets an organization as an MMO when an active subscription is detected.

In the admin interface, a new checkbox labeled `Multi-members override` has been added to the organization details page, which allows toggling this feature.

## Notes

- The `Organization` model now includes a BooleanField called `mmo_override` with a default value of `False`. This field allows superusers to override an organization's multi-member status manually.
- The Django admin form for organizations now includes a checkbox labeled `Multi-members override` that corresponds to the `mmo_override` field in the model. Checking this box will update the field and mark the organization as an MMO regardless of its subscription status.
- A new migration has been added to reflect the addition of the `mmo_override` field to the `Organization` model.

## Testing

1. Go to the Django admin page for an `organization: /admin/organizations/organization/<org_uid>/change/`
2. Verify that the new `Multi-members override` checkbox is present.
3. Toggle the checkbox and save changes. Confirm that the `mmo_override` field is updated correctly in the database.
